### PR TITLE
One Master script to rule them all.

### PR DESCRIPTION
**Pull request by C-Pablo-Diego-Rosell**
We will use this master R script for the whole Cycle 1 analytics pipeline. The Master will: 
1. Set the working directory ("setwd") for the entire workflow. 
2. Run all package installs.
3. Source all sub-scripts.

### CODE REVIEW

File: NGS2_WITNESS_Cycle1_MASTER.R [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+source ("NGS2_WITNESS_Cycle1_experiment_prep.R")
+
+### Process coloring game data
+
+source ("NGS2_WITNESS_Cycle1_coloring_prep.R") ### TO BE ADDED (Pablo)
+
+### Response funnel 
+
+source ("NGS2_WITNESS_Cycle1_response.R") ### TO BE ADDED (Matt)
```
Comment (mhoover) [inline NGS2_WITNESS_Cycle1_MASTER.R]: These `### TO BE ADDED` comments do not exist yet, correct? I would suggest that we at least comment these lines that don't exist yet to ensure a clean run if someone was to execute this. In addition, authors should be aware of the script names so we don't have a conflict down the road (i.e., I call the script something different than what is here right now).

**Comment by mhoover** [comment]
This is pretty straightforward, but would suggest action on my comment above. Fixing that up and this gets a 👍 from me.

File: NGS2_WITNESS_Cycle1_MASTER.R [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+### System information
+
+a=Sys.info()
+print(a)
+b=sessionInfo(package = NULL)
+print(a)
+
+### Set base directory
+
+setwd(X:/DARPA_NGS2/CONSULTING/Analytics/Cycle1)
```
Comment (uduwage) [inline NGS2_WITNESS_Cycle1_MASTER.R]: We should try to set this to work on multiple example.
```
a = Sys.info()[1]
if( a == "Windows") { setwd("SET THE WINDOWS PATH")}
if( a != "Windows") { setwd("SET THE WINDOWS PATH")}
```

**Review by uduwage — COMMENTED** [review]
Its a good idea that we have a main file to call all the scripts. Only one comment on setwd.

File: NGS2_WITNESS_Cycle1_MASTER.R (hunk 2 above)
Comment (C-Pablo-Diego-Rosell) [reply NGS2_WITNESS_Cycle1_MASTER.R]: Thanks Anu! I would like to make sure that we are working off of the X drive though.

Comment (mhoover) [reply NGS2_WITNESS_Cycle1_MASTER.R]: Pablo, the issue is for Mac users, there is no X drive. So, the equivalent path for Anu and I, for example, is `/Volumes/Clients/DARPA_NGS2/CONSULTING/Analytics/Cycle1`. So, to Anu's point, the following should be good:

```
a = Sys.info()[1]
if( a == "Windows") { setwd("X:/DARPA_NGS2/CONSULTING/Analytics/Cycle1")}
if( a != "Windows") { setwd("/Volumes/Clients/DARPA_NGS2/CONSULTING/Analytics/Cycle1")}
```
That way, if Anu or I were to try and execute anything we wouldn't get an error.

Comment (C-Pablo-Diego-Rosell) [reply NGS2_WITNESS_Cycle1_MASTER.R]: Thanks for clarifying Matt, will proceed with this change!

**Pull request merged**